deployments/notifyRewardAmount/optimism/givpower_distribute_extended_dec_2024.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-use-before-define */
+const hre = require("hardhat");
+const { sendReportEmail } = require("../../mailService/mailService");
+const { ethers } = hre;
+
+const pools = [
+    {
+        address: "0x301C739CF6bfb6B47A74878BdEB13f92F13Ae5E7",
+
+        // https://github.com/Giveth/giveth-dapps-v2/issues/4434
+        amount: "3825000",
```

Comment:
According to https://github.com/Giveth/giveth-dapps-v2/issues/4869
It should be: 3,584,250

Reply:
Done.